executor/show_test.go

```diff
@@ -808,6 +808,35 @@ func (s *testAutoRandomSuite) TestAutoIdCache(c *C) {
 	))
 }
 
+func (s *testAutoRandomSuite) TestAutoRandomBase(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
```

Comment:
To keep it stable, it is better to initialize the test (like other tests in `testAutoRandomSuite`):
```go
testutil.ConfigTestUtils.SetupAutoRandomTestConfig()
defer testutil.ConfigTestUtils.RestoreAutoRandomTestConfig()
```

(If you would like to help improve, these patterns can be put into `SetupTest()` and `TeardownTest()`.

Reply:
Added. Maybe I can send another PR to improve it.

Reply:
Refactored.